_content/_docs_signals/context_checks.md

```diff
@@ -95,11 +95,11 @@ An [input](inputs.md) fetches data and places it in the execution context under
 }
 ```
 
-This input executes an Elasticsearch query and stores the result of the query under the name `avg_ticket_price`.
+This input executes an Elasticsearch query and stores the result of the query under the name `avg_ticket_price`. Using the special target name `_top` places the data at the top level of the runtime data. Any data that was present before gets erased.
 
-### Accessing data in the execution context
+### Accessing runtime data
 
-Transformations, calculations and conditions access data in the execution context by using the target name:
```

Comment:
I believe using "target" here is a bit confusing, as it is not clear any more that this refers to the target of an input defined before.